docs/advanced-features/compiler.md

```diff
@@ -331,30 +361,77 @@ module.exports = {
 }
 ```
 
-Cause this code:
+The above config will transform your code as follows:
 
 ```js
+// Before
 import { MyModule } from 'my-library'
 import { App } from 'my-library/components'
 import { Header, Footer } from 'my-library/components/App'
-```
-
-To become:
 
-```js
+// After
 import MyModule from 'my-library/MyModule'
 import App from 'my-library/components/App'
 import Header from 'my-library/components/App/Header'
 import Footer from 'my-library/components/App/Footer'
 ```
 
-- Handlebars templating
+#### Using named imports
```

Comment:
Added docs for `skipDefaultConversion` ([Next.js type](https://github.com/vercel/next.js/blob/90b86be6dc50287b589363dc71a89f8bb7600736/packages/next/src/server/config-shared.ts#L507) / [SWC implementation](https://github.com/swc-project/plugins/blob/221ec0d713346c5460cae6c789f29ff9dcb07897/packages/transform-imports/transform/src/lib.rs#L84-L85)) which was not documented.

I tested this locally.